runtime/vm/jfr.cpp

```diff
@@ -814,8 +918,16 @@ jfrSamplingThreadProc(void *entryArg)
 		omrthread_monitor_enter(vm->jfrSamplerMutex);
 		vm->jfrSamplerState = J9JFR_SAMPLER_STATE_RUNNING;
 		omrthread_monitor_notify_all(vm->jfrSamplerMutex);
+		UDATA count = 0;
 		while (J9JFR_SAMPLER_STATE_STOP != vm->jfrSamplerState) {
 			J9SignalAsyncEvent(vm, NULL, vm->jfrAsyncKey);
+			if (0 == (count % 100)) { // 1000ms
+				jfrCPULoad(currentThread);
+			}
+			if (0 == (count % 1000)) { // 10s
+				J9SignalAsyncEvent(vm, NULL, vm->jfrThreadCPULoadAsyncKey);
```

Comment:
These can be nested (if count is not a multiple of 100, it's certainly not a multiple of 1000); the units can be clearer and there's no need to abbreviate:
```c
			if (0 == (count % 100)) { // 1 second
				jfrCPULoad(currentThread);
				if (0 == (count % 1000)) { // 10 seconds
					J9SignalAsyncEvent(vm, NULL, vm->jfrThreadCPULoadAsyncKey);
				}
			}
```